src/recursive_sorting/recursive_sorting.py

```diff
@@ -1,17 +1,32 @@
 # TO-DO: complete the helpe function below to merge 2 sorted arrays
-def merge( arrA, arrB ):
-    elements = len( arrA ) + len( arrB )
+def merge(arrA, arrB):
+    elements = len(arrA) + len(arrB)
     merged_arr = [0] * elements
-    # TO-DO
-    
+    combined = arrA + arrB
+
+    for index in range(0, len(merged_arr)):
+        merged_arr[index] = combined[index]
```

Comment:
idx1 = 0. # Pointer for idx of arrA
idx2 = 0 # Pointer for idx of arrB
for index in range(0, len(merged_arr)):
        # if first element in arrA is less than first element of arrB
        if arrA[idx1] < arrB[idx2]:
               # Then insert into merged_arr
              merged_arr[index] = arrA[idx1]
              idx1 += 1
         else:
                 # Case where arrA[idx1] >= arrB[idx2]:
                merged_arr[index] = arrB[idx2]
                idx2 += 1